### DISCUSSION TITLE
defaultContentLanguageInSubdir for GitHub pages

### DISCUSSION THREAD

**Discussion by dennys** [chipzoller/hugo-clarity#325] [Q&A] [answered]
It's ok to provide 2 languages "en" and "zh-tw" in my site.
And I try to enable defaultContentLanguageInSubdir = true, it's still ok in http://localhost:1313/.
But when I put it to GitHub, most functions are ok, but when I click the language switch icon, it doesn't work. (localhost is ok).
Does anybody have the same issue?

**Comment by rootwork** [comment, marked as answer]
I wonder if this is related to #324 -- @dennys are you accessing your site at an address like `dennys.github.io/projectname`?

**Reply by dennys** [reply]
I apply #324 and it works, thanks. And yes, my site is dennys.github.io, but because I'm still try lots of things, it's not stable now.

**Reply by rootwork** [reply]
OK, it's good to know that fixes this as well!
